src/web/mjs/connectors/KolNovel.mjs

```diff
@@ -9,6 +9,58 @@ export default class KolNovel extends WordPressMadaraNovel {
         this.tags = [ 'manga', 'webtoon', 'arabic' ];
         this.url = 'https://kolnovel.com';
 
-        this.novelObstaclesQuery = 'div#text-chapter-toolbar, div#ucare-quick-link-widget, div.kolno-after-content';
+        this.queryChapters = 'div.eplister ul li a';
+        this.queryChaptersTitle = 'div:not(.epl-date)';
+        this.path = '/series/list-mode/';
+
+        this.novelContentQuery = 'div.epwrapper div[itemprop="text"]';
+        this.novelObstaclesQuery = 'div[class]';
+        this.novelFormat = 'image/png';
+        this.novelWidth = '56em'; // parseInt(1200 / window.devicePixelRatio) + 'px';
+        this.novelPadding = '1.5em';
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryChapters);
+        return data.map(element => {
+            this.adLinkDecrypt(element);
+            const title = this.queryChaptersTitle ? [...element.querySelectorAll(this.queryChaptersTitle)].map(ele => ele.textContent).join(' ') : element.text;
```

Comment:
this might be able to be put in the template

Reply:
or should i just make a WordPressMangastreamNovel template (same way as madaraNovel)

Reply:
If you think the query is common and works for all websites then it needs to be confirmed by checking all websites with it. When successful it can be moved to the template.

Reply:
If you expect there will be more websites which may use the template, then create a separate template.
If you think the website just customized the WorpressMangaStream template, then don't create a separate template.

Reply:
il just leave it for now, as there are alot of WordPressMangastream connectors 
il leave the comment about WordPressMangastreamNovel encase it is a template that is reused in the future (wpthemedetector cant see kolnovel fsr)